aptos-move/framework/aptos-framework/sources/account/common_account_abstractions/common_account_abstractions_utils.move

```diff
@@ -0,0 +1,23 @@
+module aptos_framework::common_account_abstractions_utils {
+    use std::chain_id;
+    use std::string_utils;
+
+    friend aptos_framework::ethereum_derivable_account;
+    friend aptos_framework::solana_derivable_account;
+
+    public(friend) fun network_name(): vector<u8> {
```

Comment:
can you also move the tests for network_name into here?

I think we wrote tests twice, each in Solana/Ethereum